.vscode/settings.json

```diff
@@ -7,5 +7,80 @@
     },
     "dart.analysisExcludedFolders": [
         "dart-sdk"
-    ]
+    ],
+    "files.associations": {
```

Comment:
This seems like it's more of a personal setting than a project setting.

Reply:
"dart.analysisExcludedFolders": [
        "dart-sdk"
    ]
    
This was hard too find without this you see millions from Warnings in VS Code. And not the Warnings of Code in this Lib. I don't how handle it.